Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -15,14 +15,17 @@
 //******************************************************************************
 
 #import "ABContactInternal.h"
+#import "ABAddressBookManagerInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
 @implementation _ABContact
 
-- (id)initWithContact:(WACContact*)contact {
+- (id)initWithContact:(WACContact*)contact andType:(ABRecordContactType)type {
     self = [super init];
     if (self) {
         self.contact = contact;
+        self.manager = nil;
```

Comment:
but also there's no need to use `self->`

---
In reply to: [76677390](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76677390) [](ancestors = 76677390)